### ISSUE TITLE
GPU Rendering

### ISSUE THREAD

**Issue by Brooooooklyn** [Brooooooklyn/canvas#315]
https://github.com/servo/pathfinder

**Comment by stepancar** [comment]
@Brooooooklyn , I'm curious if you still have plans to support GPU?
We have a poor performance when we use shadowBlur, considering switching canvas.

**Comment by Brooooooklyn** [comment]
Yes, it's still on the roadmap, I just need some time to implement it

**Comment by Deyvan** [comment]
The easiest and fastest way would be to use EGL with OpenGL, or use Vulkan for performance. But if there will be support at least through OpenGL, I will be very happy already :)

**Comment by rambo-panda** [comment]
https://github.com/Brooooooklyn/canvas/issues/867

**Comment by Brooooooklyn** [comment]
@rambo-panda the linked issue is nothing to do with this issue, I'm fixing it in the other PR

**Comment by Deyvan** [comment]
I was able to get skia to work on vulkan
@Brooooooklyn I emailed you the code.

But forgot to tell you that you need to use vulkan headers and volk for static linking

**Comment by Brooooooklyn** [comment]
@Deyvan thanks for helping, I'm busy on other projects, so I don't have time to work on the gpu rendering features at this moment. Feel free to send me pull requests!

**Comment by rink3y** [comment]
@Brooooooklyn Is this still in the Roadmap??
